src/js/components/Clock/clock.stories.js

```diff
@@ -20,6 +45,15 @@ const AnalogClock = () => (
   </Grommet>
 );
 
+const CustomAnalogClock = () => (
+  <Grommet theme={CustomThemeAnalogClock}>
+    <Box align="center" justify="start" pad="large">
+      <Clock type="analog" />
+    </Box>
+  </Grommet>
+);
+
 storiesOf('Clock', module)
   .add('Digital Clock', () => <DigitalClock />)
-  .add('Analog Clock', () => <AnalogClock />);
+  .add('Analog Clock', () => <AnalogClock />)
+  .add('Custom Analog Clock', () => <CustomAnalogClock />);
```

Comment:
@britt6612 can you please remove all the `Clock` refs from the name of the stories?